source/grid/grid_generator.cc

```diff
@@ -4528,6 +4528,7 @@ namespace GridGenerator
 
                   c_data.vertices[j] = map_vert_index[v_index];
                   c_data.material_id = static_cast<types::material_id>(face->boundary_id());
+                  c_data.manifold_id = face->manifold_id();
```

Comment:
That's not strictly a bug fix -- just an enhancement (which makes sense). Can you add a changelog entry for this?

Reply:
sure